### PR TITLE
Plugin | Filedelete | Add support for Windows x86 with function injection method

### PR DESCRIPTION
**Pull request by skvl**

### CODE REVIEW

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test.

File: src/libinjector/injector.c [hunk 1]
```diff
@@ -537,7 +537,6 @@ bool setup_stack_64(
 }
 
 void init_argument(
```
Comment (tklengyel) [inline src/libinjector/injector.c]: I think it would make sense to just convert this function into a macro.

Comment (skvl) [reply src/libinjector/injector.c]: @tklengyel do You suggest this for optimization? If so I have made this function `inline` for [these](https://isocpp.org/wiki/faq/inline-functions#inline-vs-macros) reason.

**Comment by skvl** [comment]
@tklengyel, sorry, but I don't understand why build fails. I can successfully build it with my build machine.

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by skvl** [comment]
@tklengyel what's the problem? What should I fix?

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by tklengyel** [comment]
Thanks!

**Pull request merged**